web/template/donor/onelogin_identity_details.gohtml

```diff
@@ -0,0 +1,101 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}
+    {{ tr .App "yourConfirmedIdentityDetails" }}
+{{ end }}
+
+{{ define "main" }}
+  {{ $donor := .DonorProvided.Donor }}
+  {{ $userData := .DonorProvided.DonorIdentityUserData }}
+  {{ $donorFullName := (printf "%s %s" $userData.FirstNames $userData.LastName) }}
+  {{ $firstNamesMatch := eq $donor.FirstNames $userData.FirstNames }}
+  {{ $lastNameMatch := eq $donor.LastName $userData.LastName }}
+  {{ $dobMatch := eq $donor.DateOfBirth $userData.DateOfBirth }}
+  {{ $addressMatch := eq $donor.Address.Postcode $userData.CurrentAddress.Postcode }}
```

Comment:
would be better to inject these from the handler, since it already does this matching logic, and then we'd only need to update one place